src/marin/post_training/rl_dataset.py

```diff
@@ -202,7 +169,6 @@ def from_env_step(
         prompt_tokens = np.stack(prompt_tokens_list, axis=0)
         prompt_masks = np.stack(prompt_masks_list, axis=0)
 
-        # Compute RLOO advantages
```

Comment:
lol

Reply:
Not intentional! I added the comment back, since it's actually RLOO now. I don't like that this is all happening on the rollout worker though, I think this code should be disposed of soon.